bertopic/representation/_langchain.py

```diff
@@ -77,18 +83,60 @@ class LangChain(BaseRepresentation):
     prompt = "What are these documents about? Please give a single label."
     representation_model = LangChain(chain, prompt=prompt)
     ```
+
+    You can also use a Runnable instead of a chain.
+    The example below uses the LangChain Expression Language:
```

Comment:
Amazing! Thank you for taking the time to construct such a pipeline. This will be a perfect example of using Runnables!